### PR TITLE
14 connect 3rd party api to places controller

### PR DESCRIPTION
**Pull request by Ronapj1991**
This replaces https://github.com/ElenaCherpakova/RoR-practicum/pull/66

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +30,12 @@ Clone the repo:
...
 
     git checkout -b dev-main origin/dev-main
+
+## Creating the .env file and assigning the right API token
+
+  1. Open the ".env.example" file
+  2. In your root directory, create a file named ".env"
+  3. Open the newly created .env file
+  4. Copy the variables listed in the ".env.example" file and paste it onto the .env file
+  5. In our Google Drive: https://docs.google.com/document/u/2/ open the API Tokens file
```
Comment (dsinn) [inline README.md]: This URL just goes to the second Google account that you have linked. I suggest linking to the document itself:

```suggestion
  5. In our Google Drive, go to the [API Tokens](https://docs.google.com/document/d/1dUSup56OvHKJWdhSUIE5RjaJvZmXjGY0zBlD3qfzys8/edit?pli=1) file
```

File: README.md [hunk 2]
```diff
@@ -30,3 +30,12 @@ Clone the repo:
 Make sure to create a dev-main branch
 
     git checkout -b dev-main origin/dev-main
+
+## Creating the .env file and assigning the right API token
+
+  1. Open the ".env.example" file
```
Comment (dsinn) [inline README.md:36]: Was this file committed? I don't see it.

File: config/credentials.yml.enc
Comment (dsinn) [inline config/credentials.yml.enc:1]: I don't think this file is supposed to disappear? Rails still needs a place to store the `secret_key_base`, and now that we have a presumably private document, I suggest sharing the master key there and instructing everybody to use the same one.

File: README.md (hunk 2 above)
Comment (Ronapj1991) [reply README.md:36]: Added .env.example file

File: config/credentials.yml.enc
Comment (Ronapj1991) [reply config/credentials.yml.enc:1]: Added original credentials file back

**Review by dsinn — APPROVED** [review]

**Review by AigulY — APPROVED** [review]

**Pull request merged**